### ISSUE TITLE
Help Not Functional

### ISSUE THREAD

**Issue by bradfair** [zarthross/dokku-neo4j#2]
When I install dokku-neo4j, the dokku command becomes non-functional.  Before the plugin is installed, I can run "dokku" and see a list of potential commands.  After the plugin is installed, running "dokku" returns a syntax hint and exits.  Running a command such as "dokku apps:list" still returns valid data, however.

**Comment by zarthross** [comment]
Should be corrected by b990a2e75658e868d89cee2324cfac6ca426c9ea.

**Comment by bradfair** [comment]
Didn't seem to be :(

**Comment by zarthross** [comment]
Can you paste what the syntax hit was here?

**Comment by bradfair** [comment]
Without the plugin:

➜  ~  dokku
Usage: dokku COMMAND <app> [command-specific-options]

```
access:add                                      Add admin user
access:info <fingeprint>                        Show information about the key
access:list                                     List all SSH keys: admins and deployments
access:remove <fingerprint>                     Revoke all permissions for specific SSH key
backup:export [file]                            Export dokku configuration files
backup:import [file]                            Import dokku configuration files
config <app>                                    Display the config vars for an app
config:get <app> KEY                            Display a config value for an app
config:set <app> KEY1=VALUE1 [KEY2=VALUE2 ...]  Set one or more config vars
config:unset <app> KEY1 [KEY2 ...]              Unset one or more config vars
create <app>                                    Create shallow app
```

...

---

With the plugin:

➜  ~  dokku
Usage: dokku COMMAND <app> [command-specific-options]

**Comment by bradfair** [comment]
^^ It does not output the help text.

**Comment by zarthross** [comment]
Corrected in e1cf19d97adc94905f020e24132c3387566ef922
